ios/FluentUI/Card Nudge/CardNudgeTokens.swift

```diff
@@ -16,121 +16,151 @@ import SwiftUI
 
 /// Design token set for the `CardNudge` control.
 public class CardNudgeTokens: ControlTokens {
-    /// Creates an instance of `CardNudgeTokens` with optional token value overrides.
-    public static func create(style: MSFCardNudgeStyle,
-                              accentColor: ColorSet? = nil,
-                              accentIconSize: CGFloat? = nil,
-                              accentPadding: CGFloat? = nil,
-                              backgroundColor: ColorSet? = nil,
-                              buttonBackgroundColor: ColorSet? = nil,
-                              buttonInnerPaddingHorizontal: CGFloat? = nil,
-                              circleRadius: CGFloat? = nil,
-                              circleSize: CGFloat? = nil,
-                              cornerRadius: CGFloat? = nil,
-                              horizontalPadding: CGFloat? = nil,
-                              iconSize: CGFloat? = nil,
-                              interTextVerticalPadding: CGFloat? = nil,
-                              mainContentVerticalPadding: CGFloat? = nil,
-                              minimumHeight: CGFloat? = nil,
-                              outlineColor: ColorSet? = nil,
-                              outlineWidth: CGFloat? = nil,
-                              subtitleTextColor: ColorSet? = nil,
-                              textColor: ColorSet? = nil,
-                              verticalPadding: CGFloat? = nil) -> CardNudgeTokens {
-
-        let tokens: CardNudgeTokens
-        switch style {
-        case .standard:
-            tokens = CardNudgeTokens()
-        case .outline:
-            tokens = OutlineCardNudgeTokens()
-        }
+    public init(style: MSFCardNudgeStyle,
```

Comment:
I know we're not planning to expose this to Objc. But does it make sense to have the designated initializer internal and publish only convenience initializers?

Reply:
I did think about that, but I'm not sure what convenience initializers would look like. Ultimately there would need to be a public initializer with this same signature, at which point other convenience initializers are nice-to-have but not critical. Still, I'm open to adding a public `init(style:)` convenience initializer in addition to this one for convenience's sake (and it should be pretty tiny, binary-size-wise).